README.md

```diff
@@ -1,4 +1,37 @@
 # Premia V3 ABI
 
-This package contains Premia V3 contract deployment metadata, ABIs for the contracts used in Premia V3 SDK, and
-[Typechain](https://githubcom/dethcrypto/TypeChain) classes with ethers-v6 target.
\ No newline at end of file
+This package contains Premia V3 contract deployment metadata, ABIs for the protocol smart contracts, and [Typechain](https://githubcom/dethcrypto/TypeChain) classes with ethers-v6 target. Further details pertaining to smart contract 
+functionality can be found in our [Smart Contract Docs](https://docs-solidity.premia.finance/)
+
+Below are examples of how to access package resources in Typescript.
+
+```typescript
+import {Contract, Wallet} from 'ethers'
+// Importing of ABIs (typechain)
+import {IPool__factory} from "@premia/v3-abi/typechain";
+// Import of ABIs (json)
+import {IPool__abi} from "@premia/v3-abi/abi"
+import {IPoolFactory__abi} from "@premia/v3-abi/abi";
```

Comment:
Ignore if this was done on purpose for some reason but normally I'd have a single import for each import path.

```suggestion
import {IPool__abi, IPoolFactory__abi} from "@premia/v3-abi/abi"
```